### ISSUE TITLE
[Bug] Error when create bulk users

### ISSUE THREAD

**Issue by edenbuaa** [microsoft/pai#5511]
**OpenPAI Environment**:
- OpenPAI version: v1.7.0 or v1.3.0
- Cloud provider or hardware configuration:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Hardware (e.g. core number, memory size, storage size, GPU type etc.):
- Others:

**Anything else we need to know**:

openpai webportal report error when click 'Create Bulk Users'

![image](https://user-images.githubusercontent.com/16248697/120183566-71ee3a00-c242-11eb-9fa5-f2cab86a85d7.png)

**Comment by yiyione** [comment]
Hi @edenbuaa,
I think you click the `Create Bulk Users` button in `host/user-view.html` page.
And after click the button, will be navigated to `host/batch-register.html` page.
Can you show which page you got the error above?

**Comment by edenbuaa** [comment]
@yiyione Yes, The page nvigated to `batch-register.html` page after click the `Create Bulk users` button. But when I click the `Add New` , that error arise.

**Comment by yiyione** [comment]
OK, I got the same error, it will be fix in next release.
And workaround for the `batch-register` feature, please use the `Download template`, add the users into the csv file, and click `Browser...` button to import it.

**Comment by edenbuaa** [comment]
@Binyang2014 Is there a api to get the gpu count by sku in  vc?

**Comment by Binyang2014** [comment]
I believe there is only one SKU in a vc. So you can get the gpu count in a VC via `https://cluster-name/api/v2/virtual-clusters`

**Comment by edenbuaa** [comment]
@Binyang2014 
Maybe not, There is more  one SKUs in a vc.

**Comment by edenbuaa** [comment]
@Binyang2014 The `--cpu-manager-policy` feature  doesn't work for job pod. Is there a way to fix it?

Reproduce steps:
1. enable the `--cpu-manager-policy` feature in kubelet
2. submit a job
3.  check the cpusets allocated for pod(/var/lib/kubelet/cpu_manager_state), but no entries.

**Comment by Binyang2014** [comment]
Can you paste the full kubelet configuration? You need to set --cpu-manager-policy="static". The default value is none. BTW PAI doesn't hack for kubelet, so you will get more help from k8s community.
